tools/actioninjector.v2/internal/client/client_test.go

```diff
@@ -0,0 +1,87 @@
+package client
+
+import (
+	"context"
+	"fmt"
```

Comment:
we group imports in 3 groups and order them in this way:

std

3rd party

1st party